pkg/controller/clone-controller.go

```diff
@@ -539,8 +541,7 @@ func (r *CloneReconciler) CreateCloneSourcePod(image, pullPolicy string, pvc *co
 	util.SetRecommendedLabels(pod, r.installerLabels, "cdi-controller")
 
 	if err := r.client.Create(context.TODO(), pod); err != nil {
-		err = errors.Wrap(err, "source pod API create errored")
-		return nil, handleFailedPod(err, createCloneSourcePodName(pvc), sourcePvc, r.recorder, r.client)
+		return nil, errors.Wrap(err, "source pod API create errored")
```

Comment:
here you dont need to handle failed pod?

Reply:
this is already handled by the caller of CreateCloneSourcePod (line 287)